clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy

```diff
@@ -40,16 +40,16 @@ class BlockDeviceConfig {
     "t2.small" : [],
     "t2.medium" : [],
     "t2.large" : [],
-    "m4.large" : defaultBlockDevicesForEbsOnly(),
-    "m4.xlarge" : defaultBlockDevicesForEbsOnly(),
-    "m4.2xlarge" : defaultBlockDevicesForEbsOnly(),
-    "m4.4xlarge" : defaultBlockDevicesForEbsOnly(),
-    "m4.10xlarge" : defaultBlockDevicesForEbsOnly(),
-    "c4.large" : defaultBlockDevicesForEbsOnly(),
-    "c4.xlarge" : defaultBlockDevicesForEbsOnly(),
-    "c4.2xlarge" : defaultBlockDevicesForEbsOnly(),
-    "c4.4xlarge" : defaultBlockDevicesForEbsOnly(),
-    "c4.8xlarge" : defaultBlockDevicesForEbsOnly(),
+    "m4.large" : [],
```

Comment:
This should work. But since the `t` instances are more "off", it might be better to copy it from the `"m3.large"` config `enumeratedBlockDevicesWithVirtualName(1)` which seems to be fine (double check in AWS)?